docs/fa/class-dfa.md

```diff
@@ -309,6 +309,14 @@ for word in dfa:
     print(word)
 ```
 
+## DFA.reset_cache(self)
+
+Resets the word and count caches. Can be called if too much memory is being used.
+
+```python
+dfa.reset_word_cache()
```

Comment:
@eliotwrobson Just a note: this name does not match the actual method name in the `DFA` class.

Reply:
Fixed here: https://github.com/caleb531/automata/pull/152/commits/61e16a660cdd8bb455adf081b0bc4e9d44fb03ac